motors:
  m1:
    joint: robot_left_joint
    can_id: 1
  m2:
    joint: robot_right_joint
    can_id: 2
battery_voltage:
  - [54.18, 100]
  - [53.24, 90]
  - [53.04, 80]
  - [52.80, 70]
  - [52.65, 60]
  - [52.53, 50]
  - [51.33, 40]
  - [51.98, 30]
  - [51.55, 20]
  - [50.95, 10]
  - [45.75, 0]
battery_amperes: 15
battery_alarm_voltage: 48.5
num_inputs_per_driver: 5
num_outputs_per_driver: 3
# A/h (to simulate battery discharge)
power_consumption: 2
# A/h to simulate the battery charge
power_charge: 10
num_analog_inputs_per_driver: 1
# Values to multiply on IO raw data received
k_analog_inputs_multipliers: [6.978947368, -12.5]
voltage_analog_input_number: 1
current_analog_input_number: 2
charge_digital_output_number: 1
